File: 2021W01/JH/BOJ/10039_평균점수.java
//https://www.acmicpc.net/source/25129596

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int[] arr = new int[5];

        for(int i=0; i<arr.length; i++) {
            arr[i] = sc.nextInt();
        }

        int sum = 0;
        for(int i : arr) {
            if(i < 40) {
                sum += 40;
            } else {
                sum += i;
            }
        }

        System.out.println(sum / 5);

    }
}

File: 2021W07/Hyeseong/자유/BOJ/10039 평균 점수.cpp
// https://www.acmicpc.net/source/26610322

#include <iostream>
#include <vector>
using namespace std;

int main(void)
{
	int g;
	int t = 0;
	
	for (int i = 0; i < 5; ++i)
		{
			cin >> g;
			
			if (g < 40)
				g = 40;
			
			t += g;
		}
	
	cout << t / 5;
	
	return 0;
}